### ISSUE TITLE
Trigger automatic actions defined by auction stages in delegate server

### ISSUE THREAD

**Issue by uhbif19** [mlabs-haskell/hydra-auction#157]
This task requires to introduce polling logic placing in queue `HydraEvent`s  which are triggered by:

* `DelegateEvent Start`
* `DelegateEvent (AuctionStageStarted ...)`

Actual implementation is out of scope of this task.

Blocked by #140

**Comment by uhbif19** [comment]
@mlabs-haskell/hydra-auction

Given that our consumer is running in single thread and there may be multiple simultaneous inputs we need a way to handle one event a time.

Previously we discussed, that we may not just put everything in queue concurrently - cuz client requests were state-dependent => should use DelegateServerT, which complicates everything. But now it seems like all input producers are stateless.

**Updated:**

We should return back to concurrent threads. That is required cuz Websocket server does produce thread for each client anyway.

I guess now we could just use single `DelegateInput` queue. Responses need not to be queued - they could just be broadcasted to all clients.
